readme.md

```diff
@@ -10,8 +10,8 @@
 - Parses arguments
 - Converts flags to [camelCase](https://github.com/sindresorhus/camelcase)
 - Negates flags when using the `--no-` prefix
-- Outputs version when `--version`
-- Outputs description and supplied help text when `--help`
+- Outputs version when only the `--version`
```

Comment:
This doesn't sound good when read out loud. It's also not correct as you can have other flags together with `--version`, just not other arguments. Should we ensure there are no other flags too? Or is that too limiting? Any downsides doing it?

Reply:
I imagined a case like this (it's just an example, not the real API):

```bash
# This feature is that we could only type `package-json` under a package,
# it will read package.json file automatically then return the result.

~/Git/npm foo-branch ⇡
❯ package-json --version=next --full-metadata --all-versions

{
	"name": "npm",
	"version": "6.9.0-next.0"
	...
}
```

In this case, we don't have any `cli.input` but have three flags for the command.

I think we should ensure there are no other flags and inputs for the version output. It should not be output even `version` has a value.

Reply:
There are too many logics to explain for this issue. We could add a new section for document the issue, or document in the `autoVersion` and `autoHelp` section.

Reply:
> I think we should ensure there are no other flags and inputs for the version output. It should not be output even `version` has a value.

:+1:

> We could add a new section for document the issue

👍